integrations/newrelic/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "newrelic"
-version = "0.1.0"
+version = "0.1.2"
```

Comment:
Why 2 patch versions bump?

Reply:
Tom forgot to update this file along with spec.yml